submissions/kovalov/dom/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Fantasy Planets</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__container container">
+        <a href="./" class="header__link">Planets </a>
+        <button class="header__button" data-menu-button>
+          <svg class="header__button-icon">
+            <use href="./assets/sprite.svg#menu-icon"></use>
+          </svg>
+        </button>
+        <nav
+          class="header__navigation navigation"
+          data-mobile-menu-content
+        >
+          <ul class="navigation__list">
+            <li class="navigation__list-item">
+              <a href="#" class="navigation__link" data-id="0"
+                >Tatooine</a
+              >
+            </li>
+            <li class="navigation__list-item">
+              <a href="#" class="navigation__link" data-id="1"
+                >Vulcan</a
+              >
+            </li>
+            <li class="navigation__list-item">
+              <a href="#" class="navigation__link" data-id="2"
+                >Xandar</a
+              >
+            </li>
+            <li class="navigation__list-item">
+              <a href="#" class="navigation__link" data-id="3"
+                >Krypton</a
+              >
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <div class="content">
+      <div class="content__container container">
+        <aside class="aside">
+          <nav
+            class="aside__navigation navigation"
+            data-desktop-menu-content
+          >
+            <ul class="navigation__list">
+              <li class="navigation__list-item">
```

Comment:
If you change something in your data object you need to make changes there too, because you have a `data-id` attribute value linked to your data object, think about how you can avoid this situation.